<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="18" passed="0" failed="18" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-18T18:54:18 IST" name="Suite" finished-at="2024-12-18T18:55:28 IST" duration-ms="69868">
    <groups>
      <group name="PL1">
        <method signature="RestAssured_TestCases_PL1.getAllDepartmentsTest()[pri:1, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getAllDepartmentsTest" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getAllItemsTest()[pri:2, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getAllItemsTest" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getIncentiveSummary()[pri:3, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getIncentiveSummary" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getIncentiveReffSummary()[pri:4, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getIncentiveReffSummary" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getHospitalIncomeIncReport()[pri:5, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getHospitalIncomeIncReport" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getIncenEmpBillItems()[pri:6, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getIncenEmpBillItems" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.getInventoryFiscalYears()[pri:7, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="getInventoryFiscalYears" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.activateInventory()[pri:8, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="activateInventory" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.inventorySubCategory()[pri:9, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="inventorySubCategory" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.availableItems()[pri:10, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="availableItems" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.requisitionItems()[pri:11, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="requisitionItems" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.verifyRequisitionAndDispatchFields()[pri:12, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="verifyRequisitionAndDispatchFields" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.inventoryItemByStoreId()[pri:13, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="inventoryItemByStoreId" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.mostSoldMedicine()[pri:14, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="mostSoldMedicine" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.substoreWiseDisp()[pri:15, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="substoreWiseDisp" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.activSupp()[pri:16, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="activSupp" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.measureUnits()[pri:17, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="measureUnits" class="testcases.RestAssured_TestCases_PL1"/>
        <method signature="RestAssured_TestCases_PL1.salesCat()[pri:18, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" name="salesCat" class="testcases.RestAssured_TestCases_PL1"/>
      </group> <!-- PL1 -->
    </groups>
    <test started-at="2024-12-18T18:54:18 IST" name="Test" finished-at="2024-12-18T18:55:28 IST" duration-ms="69868">
      <class name="testcases.RestAssured_TestCases_PL1">
        <test-method signature="getAllDepartmentsTest()[pri:1, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:18 IST" name="getAllDepartmentsTest" description="1. Send a GET request to Get All Departments
2. Validate that all the DepartmentId and DepartmentName are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:19 IST" duration-ms="432" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getAllDepartments must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getAllDepartments must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getAllDepartmentsTest(RestAssured_TestCases_PL1.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllDepartmentsTest -->
        <test-method signature="getAllItemsTest()[pri:2, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:28 IST" name="getAllItemsTest" description="1. Send a GET request to Get All Items
2. Validate that the Item Id and Item name are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:28 IST" duration-ms="9" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getAllItems must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getAllItems must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getAllItemsTest(RestAssured_TestCases_PL1.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllItemsTest -->
        <test-method signature="getIncentiveSummary()[pri:3, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:31 IST" name="getIncentiveSummary" description="1. Send a GET request to Incentive Summary Report
2. Validate that the Prescriber Id and Prescriber name are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:35 IST" duration-ms="3314" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Status code should be 200 OK. expected [200] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Status code should be 200 OK. expected [200] but found [0]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at testcases.RestAssured_TestCases_PL1.getIncentiveSummary(RestAssured_TestCases_PL1.java:127)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIncentiveSummary -->
        <test-method signature="getIncentiveReffSummary()[pri:4, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:38 IST" name="getIncentiveReffSummary" description="1. Send a GET request to Incentive Referral Summary Report
2. Validate that the Prescriber name, PrescriberId, DocTotalAmount, TDSAmount and NetPayableAmount are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:38 IST" duration-ms="696" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getIncentiveReffSummary must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getIncentiveReffSummary must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getIncentiveReffSummary(RestAssured_TestCases_PL1.java:164)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIncentiveReffSummary -->
        <test-method signature="getHospitalIncomeIncReport()[pri:5, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:42 IST" name="getHospitalIncomeIncReport" description="1. Send a GET request to Hospital Income Incentive Report
2. Validate that the ServiceDepartmentName, ServiceDepartmentId, NetSales, ReferralCommission, GrossIncome, OtherIncentive, and HospitalNetIncome are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:42 IST" duration-ms="202" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getHospIncIncReport must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getHospIncIncReport must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getHospitalIncomeIncReport(RestAssured_TestCases_PL1.java:233)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getHospitalIncomeIncReport -->
        <test-method signature="getIncenEmpBillItems()[pri:6, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:45 IST" name="getIncenEmpBillItems" description="1. Send a GET request to Incentive Employee Bill Items
2. Validate that the EmployeeIncentiveInfoId, EmployeeId, FullName, TDSPercent, EmpTDSPercent, IsActive are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:45 IST" duration-ms="104" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getIncenEmpBillItems must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getIncenEmpBillItems must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getIncenEmpBillItems(RestAssured_TestCases_PL1.java:313)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIncenEmpBillItems -->
        <test-method signature="getInventoryFiscalYears()[pri:7, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:48 IST" name="getInventoryFiscalYears" description="1. Send a GET request to Inventory Fiscal Years
2. Validate that the FiscalYearId, FiscalYearName are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:49 IST" duration-ms="21" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getInvntryFiscalYrs must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getInvntryFiscalYrs must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.getInventoryFiscalYears(RestAssured_TestCases_PL1.java:384)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getInventoryFiscalYears -->
        <test-method signature="activateInventory()[pri:8, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:52 IST" name="activateInventory" description="1. Send a GET request to Activate Inventory/
2. Validate that the StoreId, Name, and StoreDescription are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:52 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getActInventory must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getActInventory must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.activateInventory(RestAssured_TestCases_PL1.java:445)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- activateInventory -->
        <test-method signature="inventorySubCategory()[pri:9, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:55 IST" name="inventorySubCategory" description="1. Send a GET request to Inventory Subcategory
2. Validate that the SubCategoryName and SubCategoryId are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:55 IST" duration-ms="12" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getInvSubCat must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getInvSubCat must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.inventorySubCategory(RestAssured_TestCases_PL1.java:500)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inventorySubCategory -->
        <test-method signature="availableItems()[pri:10, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:54:58 IST" name="availableItems" description="1. Send a GET request to available Items by Store Id
2. Validate that the ItemId, AvailableQuantity and StoreId are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:54:58 IST" duration-ms="189" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[availableItems must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: availableItems must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.availableItems(RestAssured_TestCases_PL1.java:548)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- availableItems -->
        <test-method signature="requisitionItems()[pri:11, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:01 IST" name="requisitionItems" description="1. Send a GET request to get Inventory Requisition Items For View by requisition Id.
2. Validate that the RequisitionNo, CreatedByName are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:01 IST" duration-ms="99" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[requisitionItems must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: requisitionItems must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.requisitionItems(RestAssured_TestCases_PL1.java:605)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- requisitionItems -->
        <test-method signature="verifyRequisitionAndDispatchFields()[pri:12, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:05 IST" name="verifyRequisitionAndDispatchFields" description="1. Send a GET request to track by requisition Id.
2. Validate that the RequisitionId, CreatedBy and Dispatchers are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:05 IST" duration-ms="97" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[verifyRequisitionAndDispatchFields must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: verifyRequisitionAndDispatchFields must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.verifyRequisitionAndDispatchFields(RestAssured_TestCases_PL1.java:686)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRequisitionAndDispatchFields -->
        <test-method signature="inventoryItemByStoreId()[pri:13, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:08 IST" name="inventoryItemByStoreId" description="1. Send a GET request to get inventory items by store Id.
2. Validate that the ItemId, ItemName, and AvailableQuantity are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:08 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getInvItem must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getInvItem must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.inventoryItemByStoreId(RestAssured_TestCases_PL1.java:757)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inventoryItemByStoreId -->
        <test-method signature="mostSoldMedicine()[pri:14, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:11 IST" name="mostSoldMedicine" description="1. Send a GET request to get most sold medicine within date range.
2. Validate that the ItemName and SoldQuantity are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:12 IST" duration-ms="176" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getMostSoldMed must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getMostSoldMed must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.mostSoldMedicine(RestAssured_TestCases_PL1.java:830)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mostSoldMedicine -->
        <test-method signature="substoreWiseDisp()[pri:15, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:15 IST" name="substoreWiseDisp" description="1. Send a GET request to get substore wise dispatch value.
2. Validate that the Name and TotalDispatchValue are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:15 IST" duration-ms="101" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getSubDisp must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getSubDisp must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.substoreWiseDisp(RestAssured_TestCases_PL1.java:893)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- substoreWiseDisp -->
        <test-method signature="activSupp()[pri:16, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:18 IST" name="activSupp" description="1. Send a GET request to get active suppliers.
2. Validate that the SupplierId and SupplierName are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:18 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getActSupp must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getActSupp must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.activSupp(RestAssured_TestCases_PL1.java:947)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- activSupp -->
        <test-method signature="measureUnits()[pri:17, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:21 IST" name="measureUnits" description="1. Send a GET request to get unit of measurements.
2. Validate that the UOMId and UOMName are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:21 IST" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getMeasureUnits must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getMeasureUnits must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.measureUnits(RestAssured_TestCases_PL1.java:996)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- measureUnits -->
        <test-method signature="salesCat()[pri:18, instance:testcases.RestAssured_TestCases_PL1@65fe9e33]" started-at="2024-12-18T18:55:25 IST" name="salesCat" description="1. Send a GET request to get sales categories from pharmacy.
2. Validate that the Name and SalesCategoryId are not null.
3. Verify the response status code is 200." finished-at="2024-12-18T18:55:25 IST" duration-ms="13" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[getSalesCat must be implemented using Rest Assured methods only. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: getSalesCat must be implemented using Rest Assured methods only. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testcases.RestAssured_TestCases_PL1.salesCat(RestAssured_TestCases_PL1.java:1037)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- salesCat -->
      </class> <!-- testcases.RestAssured_TestCases_PL1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
